packages/runtime/container-runtime/src/id-compressor/idCompressor.ts

```diff
@@ -283,15 +285,13 @@ export class IdCompressor {
 
 	/**
 	 * The size of each newly created ID cluster.
+	 * Must only be set with a value upon which consensus has been reached. Value must be greater than zero and less than
```

Comment:
Why move these docs? IIRC, intellisense automatically concats them into one doc for both.

Reply:
At some point I got an api-extractor error/warning for this but looks like it's not there any more :thinking:. In any case looking at the api-extractor output there's quite a few warnings I need to clean up.